<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VueMenuJeu</class>
 <widget class="QWidget" name="VueMenuJeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="LayoutMenuJeu">
     <item>
      <spacer name="spacerGauche">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="contenuLayout">
       <item>
        <spacer name="spacerHaut">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelProfilNom">
         <property name="text">
          <string>Nom Du Profil</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boutonNouvellePartie">
         <property name="text">
          <string>Nouvelle Partie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boutonChargerPartie">
         <property name="text">
          <string>Charger Partie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boutonCreerPartieMultijoueur">
         <property name="text">
          <string>Cr√©er Partie Multijoueur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boutonRejoindrePartieMulti">
         <property name="text">
          <string>Rejoindre Partie Multijoueur</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerBas">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacerDroit">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
